src/main/jhipster/src/main/webapp/app/germplasm-manager/germplasm-search.component.ts

```diff
@@ -807,11 +807,6 @@ export class GermplasmSearchComponent implements OnInit {
             return;
         }
 
-        if (this.size(this.selectedItems) > 500) {
```

Comment:
Check the this.germplasmService.deleteGermplasm() method in typescript. If a user selected a large number of items to delete (e.g. 2000 records), A "Request header is too large" will occur.  @cheryleigh